graylog2-web-interface/src/views/components/SearchBar.jsx

```diff
@@ -80,18 +81,23 @@ const SearchBar = ({ availableStreams, config, currentQuery, disableSearch = fal
                                     config={config} />
                   </Col>
 
-                  <Col md={6}>
+                  <Col mdHidden lgHidden>
+                    <HorizontalSpacer />
+                  </Col>
```

Comment:
As you can see in the screenshots, one problem is the spacing between the Col's when they get the full width. I thought about adding a class to the specific column and controlling its bottom margin with a media query. The problem is, we can't access bootstraps media query points and would have to hard code them here and in every other comparable scenario.
It feels a bit strange to have a component only for some spacing. But this way we can use bootstraps col properties `mdHidden lgHidden` and in comparison it is more readable.

Reply:
I am cool with that solution. I like the idea! :+1: